archive/f/frege/README.md

```diff
@@ -0,0 +1,19 @@
+# Sample Programs in Frege
+
+Welcome to Sample Programs in Frege!
+
+## Sample Programs
+
+- [Hello World in Frege][1]
+
+## Fun Facts
+
+- Debut: 2011
+- Typing: Static
+
+## References
+
+- [Frege Wiki][2]
+
+[1]: https://github.com/TheRenegadeCoder/sample-programs/issues/734
```

Comment:
Looks great! Can you create a new article issue like #400 and link it here instead?

Reply:
Sure

Reply:
I took care of it!